tests/openbis_functions/test_openbis_db.py

```diff
@@ -0,0 +1,55 @@
+        mock_df = pd.DataFrame.from_dict(pybis_response)
+        return mock_df
+
+
+@pytest.fixture
+def mock_get_sample_df(monkeypatch):
+
+    def mock_get(*args, **kwargs):
+        return MockResponseGetSampleNames()
+
+    monkeypatch.setattr(Openbis, "get_samples", mock_get)
+
+
+def test_get_sample_names(mock_get_sample_df):
+
+    o = Openbis("https://fakeurl")
+    space = 'CKUJATH'
+    project = 'TEST_AMBETON'
+    collection = '/CKUJATH/TEST_AMBETON/VISKO_DATA_COLLECTION'
+
+    # app.get_json, which contains requests.get, uses the monkeypatch
+
+    result = ExpStep.get_sample_names(o, space, project, collection)
```

Comment:
In this case not much, the get_sample_names is a very simple method. I wanted to keep the mocking in case i wanted to write more mocked tests, but that problem may be solved by running the openBIS image on actions.